src/ai-did-you-mean-this/README.rst

```diff
@@ -0,0 +1,6 @@
+Microsoft Azure CLI 'AI Did You Mean This' Extension
+==========================================
+
+Improve user experience by suggesting recovery options for common CLI failures.
+
+This extension extends the default error handling behavior to include recommendations for recovery. Recommendations are based on how other users were successful after they encountered the same failure. 
```

Comment:
How about adding some instructions and usage scenarios? Like what kind of error can trigger this extension.